includes/class-db.php

```diff
@@ -41,8 +41,9 @@ public static function update_database() {
 
 		$version_from_db = self::get_version();
 
-		if ( version_compare( $version_from_db, '1.0.0', '<' ) ) {
+		if ( version_compare( $version_from_db, '1.0.1', '<' ) ) {
```

Comment:
Please add a different block for 1.0.1 and keep the 1.0.0, so that we can see what changed over time.